content/doc/developer/publishing/_chapter.yml

```diff
@@ -17,5 +17,7 @@ guides:
 #- adopt-a-plugin
 #- changing-maintainers
 - releasing
+- releasing-cd
+- releasing-manually
```

Comment:
```suggestion
```

IMO not every page needs to be linked from the sidepanel as long as there is a good landing page, which we have.